commit 4ec4888593f739328600d42c1ad5a33d6b72674a
Author: Vratislav Podzimek <vpodzime@redhat.com>
Date:   Tue Apr 8 09:37:01 2014 -0400

    Remove extra del(fo) for weird C NULL execption (it is valid). BZ 1058297.

diff --git a/yum/misc.py b/yum/misc.py
index 347a07a..cdc08a0 100644
--- a/yum/misc.py
+++ b/yum/misc.py
@@ -360,7 +360,6 @@ def checksum(sumtype, file, CHUNK=2**16, datasize=None):
 
         if type(file) is types.StringType:
             fo.close()
-            del fo
             
         # This screws up the length, but that shouldn't matter. We only care
         # if this checksum == what we expect.
